src/Symfony/Component/Validator/Mapping/Cache/Psr6Cache.php

```diff
@@ -66,6 +66,6 @@ public function write(ClassMetadata $metadata)
      */
     private function escapeClassName(string $class): string
     {
-        return str_replace('\\', '.', $class);
+        return rawurlencode(str_replace('\\', '.', $class));
```

Comment:
let's remove str_replace ?

Reply:
Fixed

Reply:
Tests failed because of this reverted change.